etl/pad/storage/dungeon.py

```diff
@@ -203,3 +202,46 @@ def __init__(self,
 
     def __str__(self):
         return 'SDRewardData({}): {}'.format(self.key_value(), self.reward_na)
+
+
+class FixedCard(ServerDependentSqlItem):
+    """A fixed card in a subdungeon fixed team."""
```

Comment:
same deal with the class name here.

For the table name actually I'd like to call it fixed_team_monsters & fixed_teams as the two tables, if that's ok

that'll make them alphabetize exactly next to each other & also be precise about what they are & also remove the need to introduce the "card" terminology